README.md

```diff
@@ -25,28 +82,218 @@ This repository contains extension code for SQL Server input and output bindings
 1. Install the extension.
 
     ```bash
-    dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --version 1.0.0-preview1
+    dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --version 1.0.0-preview2
```

Comment:
once I upload it, this will be `1.0.0-preview3`

Reply:
Will update and test